### PR TITLE
use simpler 'release' instead of 'non-prerelease' word in user-facing…

### PR DESCRIPTION
**Pull request by pastey**
… output

Hi,

Not sure if you'll find this PR useful and corresponding to your philosophy, but anyway...

I find current phrasing `Latest non-prerelease version available is...` a bit too complex. I would suggest to simplify it by replacing `non-prerelease` to plain... `release`.

I see that the same `NonPrerelease` term is used in variable names. If you find this suggestion feasible, then I can rename variables too.

Maybe you'll tell me that there's some deep idea behind `non-prerelease`, then I'm fine with that too :)

Thanks for the great tool that saves ton of my time!

### CODE REVIEW

File: Sources/XcodesKit/XcodeInstaller.swift [hunk 1]
```diff
@@ -62,7 +62,7 @@ public final class XcodeInstaller {
             case let .unavailableVersion(version):
                 return "Could not find version \(version.appleDescription)."
             case .noNonPrereleaseVersionAvailable:
```
Comment (MattKiazyk) [inline Sources/XcodesKit/XcodeInstaller.swift:64]: Can't agree with you more! Can we go the next step and rename this enum as well?

**Review by MattKiazyk — CHANGES_REQUESTED** [review]
I aggee @pastey - can we change the enum as well so devs working on the project are less confused too!

File: README.md [hunk 2]
```diff
@@ -147,7 +147,7 @@ You'll need Xcode 13 in order to build and run xcodes.
 
 <details>
 <summary>Using Xcode</summary>
-Even though xcodes is a command-line app, lll of the normal functionality works in Xcode, like building, running, and running tests. You can even type text into Xcode's console when it prompts you for input like your Apple ID or 2FA code.
+Even though xcodes is a command-line app, all of the normal functionality works in Xcode, like building, running, and running tests. You can even type text into Xcode's console when it prompts you for input like your Apple ID or 2FA code.
```
Comment (pastey) [inline README.md:150]: Fixed a small typo here. Not related to the PR, but would be nice to fix 🙏

**Comment by pastey** [comment]
@MattKiazyk, please check 🙏

**Comment by MattKiazyk** [comment]
Sorry @pastey can you rebase with the recent changes with runtime installs. Looks good otherwise.

**Comment by pastey** [comment]
> Sorry @pastey can you rebase with the recent changes with runtime installs. Looks good otherwise.

Done. Hope I did it correctly (I'm more used to plain old merge)

**Comment by pastey** [comment]
Hey, @MattKiazyk will you have a chance to take one more look at this PR?

**Comment by pastey** [comment]
Hey, @MattKiazyk will you have a chance to take one more look at this PR?

**Review by MattKiazyk — APPROVED** [review]

**Pull request merged**